samples/azure/azure-service-fabric-routing/Core_6/CandidateVoteCount/CandidateVotes.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Threading.Tasks;
+using NServiceBus;
+
+public class CandidateVotes :
+    Saga<CandidateVotes.CandidateVoteData>,
+        IAmStartedByMessages<VotePlaced>,
+        IHandleMessages<CloseElection>,
+        IHandleMessages<TrackZipCodeReply>
+{
+
+    protected override void ConfigureHowToFindSaga(SagaPropertyMapper<CandidateVoteData> mapper)
+    {
+        mapper.ConfigureMapping<VotePlaced>(m => m.Candidate)
+            .ToSaga(s => s.Candidate);
+        mapper.ConfigureMapping<CloseElection>(m => m.Candidate)
+            .ToSaga(s => s.Candidate);
+    }
+
+    public Task Handle(VotePlaced message, IMessageHandlerContext context)
+    {
+        if (!Data.Started)
+        {
+            Data.Candidate = message.Candidate;
+            Data.Started = true;
+        }
+        Data.Count++;
+
+        var trackZipCode = new TrackZipCode
+        {
+            ZipCode = message.ZipCode
+        };
+        return context.Send(trackZipCode);
+    }
+
+    public async Task Handle(CloseElection message, IMessageHandlerContext context)
+    {
+        var reportVotes = new ReportVotes
+        {
+            Candidate = Data.Candidate,
+            NumberOfVotes = Data.Count
+        };
+        await context.SendLocal(reportVotes)
```

Comment:
@tmasternak Ok, so normally SendLocal would be wrong here from my perspective but you made the following comment elsewhere:

> The intention here was that local sends and reply-to are partition stick

Meaning, you want this send to be routed to any destination within the partition but still to the same endpoint. That is strange.

SHouldn't we then have a partition sendoption?

    var options = new SendOption()
    options.RouteToCurrentPartition();
    await context.Send(reportVotes, options);


This to me seems like abusing `SendLocal` for another intent.